[IMP] Employees: updating new employee doc for version 17 (#7429)
odoo/documentation: content/applications/hr/employees/new_employee.rst

is a centralized place where all important information about the employee is stored, including
:ref:`general information <employees/general-info>`, :ref:`job history and skills
<employees/resume>`, :ref:`various work information <employees/work-info-tab>`, :ref:`personal
details <employees/private-info>`, :ref:`documents <employees/docs>`, and more.

Starting in the :menuselection:`Employees` app dashboard, click the :guilabel:`New` button and a new
employee form appears. Fill out the required information and any additional details.

.. image:: new_employee/new-employee-form.png
   :align: center
   :alt: Create a new employee form with all fields filled out.

.. note::
   The current company phone number and name is populated in the :guilabel:`Work Phone` and
   :guilabel:`Company` fields. If the *Appraisals* application is installed, the :guilabel:`Next
   Appraisal Date` field is populated with a date six months from today's date.

.. _employees/general-info:

General information
===================

The employee form automatically saves as data is entered, but the save manually icon (a cloud with

> an up arrow inside it) may be clicked at any time to manually save the form.

Author: Sounds good! I really wish that icon was something less descriptive, or just said 'save' so it wasn't such a lengthy description every time we mention the button =D  I'll add this to my notes so I am consistent.

Required fields
---------------

- :guilabel:`Name`: enter the employee's name.
- :guilabel:`Company`: select the company from the drop-down menu that the new employee is hired by,

>   or create a new company by typing the name in the field.

Author: Just changed it to :guilabel:`Create` or :guilabel:`Create and edit...`

.. note::
   The current company phone number and name is populated in the :guilabel:`Work Phone` and

>    :guilabel:`Company` fields. If the *Appraisals* application is installed, the :guilabel:`Next

Author: I added a brief summary and made anchors, linking each item in the list to the anchor- I think that's a much better intro- thank you!

   Appraisal Date` field is populated with a date six months from today's date.

General information
===================

> The employee form automatically saves as data is entered, but the save manually icon (a cloud with

Author: I only disagree with this since the cloud does not have the arrow in it, and I think that is a very important part of the description, and the icon might confuse the reader. Since I need to add the text abut the arrow, I will keep it all text.

Optional fields
---------------

- :guilabel:`Photo`: in the top right image box of the employee form, click on the :guilabel:`✏️
  (pencil)` edit icon to select a photo to upload.
- :guilabel:`Job Position`: enter the employee's job title under their name or select it from the
  :guilabel:`Job Position` drop-down menu to have this field auto-populate. The :guilabel:`Job
  Position` field under the employee name can be modified and does not need to match the selection
  made in the :guilabel:`Job Position` drop-down.

  .. example::
     While it is recommended to have the job positions match, the typed-in description in this field
     can contain more information than the selected drop-down job position if desired.

     For instance, if someone is hired for a sales representative position configured as
     :guilabel:`Sales Representative` in the Recruitment app, that can be selected in the drop-down
     :guilabel:`Job Position`` field.

     In the typed in job position field beneath the employee's name, the position could be more

>      specific, such as `Sales Representative - Subscriptions` if the employee is focused solely on

Author: I am a little confused since I see it in the green text in the suggestion block right above? I made the edits to match what I see in the green. If this is not what you meant, then please let me know.

     field can contain more specific information than the selected drop-down :guilabel:`Job
     Position`, if desired.

     For instance, if someone is hired for a sales representative position configured as
     :guilabel:`Sales Representative` in the *Recruitment* app, that can be selected in the
     drop-down :guilabel:`Job Position` field.

     In the typed-in :guilabel:`Job Position` field beneath the :guilabel:`Employee's Name` field,
     the position could be more specific, such as `Sales Representative - Subscriptions` if the
     employee is focused solely on subscription sales.

     .. image:: new_employee/job-description-fields.png
        :align: center
        :alt: Both job position fields entered but with different information.

- :guilabel:`Tags`: select a tag from the drop-down menu to add relevant tags to the employee. Any
  tag can be created in this field by typing it in. Once created, the new tag is available for all
  employee records. There is no limit to the amount of tags that can be added.
- Work Contact Information: enter the employee's :guilabel:`Work Mobile`, :guilabel:`Work Phone`,

>   :guilabel:`Work Email`, and/or :guilabel:`Company` name, if not already auto-populated.

Author: I totally agree with this! I just never made it a GUI because technically, it does not get one. I HAPPILY am changing this since I agree, it looks better. I just didn't think I was "allowed" to since it is technically incorrect.

        :align: center
        :alt: Both job position fields entered but with different information.

> - Tags: click on a tag in the drop-down menu to add any tags applicable to the employee. Any tag can

Author: I am adding it because it looks better with al of them guilabeled.  My logic for not using a gui was that the word isn't bolded, and it disappears after you select a tag. The name is more of a placeholder and disappears after yo assign the tags. But I am changing it to a gui, which looks better overall.

  be created in this field by typing it in. Once created, the new tag is available for all employee
  records. There is no limit to the amount of tags that can be added.

> - Work Contact Information: Enter the employee's :guilabel:`Work Mobile`, :guilabel:`Work Phone`,

Author: Weird- I see the word "click" as lower-case in my doc already, so not sure why it doesn't appear that way.

As for the work info, it should not because that's just a phrase used to describe the section where the work info is housed. That phrase does not appear anywhere on the form, and is a way for me to describe the left side of the card, in the top section.

  :guilabel:`Work Email`, and/or :guilabel:`Company` name.
- :guilabel:`Department`: select the employee's department from the drop-down menu.
- :guilabel:`Job Position`: select the employee's job position from the drop-down menu. Once a
  selection is made, the :guilabel:`Job Position` field beneath the employee's name automatically
  updates to reflect the currently selected job position. These positions are from the
  :doc:`Recruitment <../../hr/recruitment/new_job/>` application, and reflect the currently \
  configured job positions.
- :guilabel:`Manager`: select the employee's manager from the drop-down menu.
- :guilabel:`Coach`: select the employee's coach from the drop-down menu.
- :guilabel:`Next Appraisal Date`: this field is only visible if the *Appraisals* application is
  installed. The date automatically populates with a date that is computed according to the settings

>   configured in the *Appraisals* application. This date can be modified using the calendar selector.

Author: The document is being written now so it is not published yet/there is no link to place here right now. But I can edit this doc later once it is live and add it.

Additional information tabs
===========================

.. _employees/resume:

> Resumé tab

Author: I have updated the wording for the form, but I left the accent marks. The reason why, is Zac had made the edit originally in the ealrier version of these docs, and it appears in the live doc currently:
![image](https://github.com/odoo/documentation/assets/121518652/602a10c4-f132-41c3-9b7d-f39a82d21834)

So since it shows, and is more accurate, and Zac requested it originally, I am going to leave this for now.

  :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,
  :guilabel:`Completed Internal Training`, or type in a new entry, then click :guilabel:`Create
  "(Type)"`.
- :guilabel:`Display Type`: From the drop-down menu, choose :guilabel:`Classic` for typical work
  experience, :guilabel:`Certification` for experience gained through a certificate, or
  :guilabel:`Course` for non-certified classes.
- :guilabel:`Duration`: enter the start and end dates for the work experience. To select a date, use
  the :guilabel:`< (left arrow)` and :guilabel:`> (right arrow)` icons to scroll to the desired
  month, then click on the day to select it.
- :guilabel:`Description`: enter any relevant details in the field.

Once all the information is entered, click the :guilabel:`Save & Close` button if there is only one
entry to add, or click the :guilabel:`Save & New` button to save the current entry and create
another resumé line.

.. note::
   After the new employee form is saved, the current position and company is automatically added to
   the :guilabel:`Resumé` tab, with the end date listed as `current`.

> Skills

Author: Same for the previous comment- Zac had made the original change with the accent, so I did not want to remove it. I think it might be best to stay consistent, even though the buttons do not have the accent. I will leave this for him to decide in the final review (or whoever is doing the final review).

       :guilabel:`Administrator` set for the :guilabel:`Payroll` role.

- :guilabel:`Remote Work`: use the drop-down menu to select the default location the employee works
  from each day of the week. The default options are :guilabel:`Home`, :guilabel:`Office`, or
  :guilabel:`Other`.

  A new location can be typed into the field, then click either :guilabel:`Create (new location)` to
  add the location, or :guilabel:`Create and edit...` to add the new location and edit the form.

  After edits are done, click :guilabel:`Save & Close`, and the new location is added, and populates
  the field.

  Leave the field blank (:guilabel:`Unspecified`) for non-working days like Saturday and Sunday.

  .. note::
     It is also possible to add or modify work locations by navigating to :menuselection:`Employees
     app --> Configuration --> Employee: Work Locations`. To modify a location, click on an existing
     location, then make any changes on the form.

     Click :guilabel:`New` to create a new location, then enter the following information on the
     form. All fields are **required**.

     - :guilabel:`Work Location`: enter the name for the location. This can be as general or

>        specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.

Reviewer: ```suggestion
       as specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.
```

located. Different fields are configured for different locations, however some sections appear
regardless.

- :guilabel:`Status`: select an :guilabel:`Employee Type` and, if applicable, a :guilabel:`Related
  User`, with the drop-down menus. The :guilabel:`Employee Type` options include
  :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
  :guilabel:`Freelancer`.

  .. important::
     Employees do **not** also need to be users. *Employees* do **not** count towards the Odoo
     subscription billing, while *Users* **do** count towards billing. If the new employee should
     also be a user, the user **must** be created.

     After the employee is created, click the :guilabel:`⚙️ (gear)` icon, then click
     :guilabel:`Create User`. A :guilabel:`Create User` form appears.

     Type in the :guilabel:`Name` and :guilabel:`Email Address`. Next, select the
     :guilabel:`Company` from the drop-down menu.

     Then, enter the :guilabel:`Phone` and :guilabel:`Mobile` numbers in the respective fields.

     If a photo is available, click the :guilabel:`Edit` icon (which appears as a :guilabel:`✏️

>      (pencil)` icon) in the lower-left corner of the image box, which is located in the top-right of
>      the form.

Reviewer: ```suggestion
     (pencil)` icon) in the lower-left corner of the image box, which is located in the top-right
     corner of the form.
```

     A file explorer pops up. Navigate to the file, then click :guilabel:`Open` to select it.
     Finally, click :guilabel:`Save` after all the information is entered, and the employee record

>      is automatically updated with the newly created user populating the :guilabel:`Related User

Reviewer: ```suggestion
     is automatically updated with the newly-created user populating the :guilabel:`Related User
```

       either :guilabel:`Manager`, :guilabel:`Officer:Manage all contracts`, or
       :guilabel:`Administrator` set for the :guilabel:`Payroll` role.

- :guilabel:`Remote Work`: use the drop-down menu to select the default location the employee works
  from each day of the week. The default options are :guilabel:`Home`, :guilabel:`Office`, or
  :guilabel:`Other`.

  A new location can be typed into the field, then click either :guilabel:`Create (new location)` to
  add the location, or :guilabel:`Create and edit...` to add the new location and edit the form.

  After edits are done, click :guilabel:`Save & Close`, and the new location is added, and populates
  the field.

  Leave the field blank (:guilabel:`Unspecified`) for non-working days like Saturday and Sunday.

  .. note::
     It is also possible to add or modify work locations by navigating to :menuselection:`Employees
     app --> Configuration --> Employee: Work Locations`. To modify a location, click on an existing
     location, then make any changes on the form.

     Click :guilabel:`New` to create a new location, then enter the following information on the
     form. All fields are **required**.

>      - :guilabel:`Work Location`: enter the name for the location. This can be as general or
>        as specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.

Reviewer 2: Early break
```suggestion
     - :guilabel:`Work Location`: enter the name for the location. This can be as general or as
       specific, as needed, such as `Home` or `Building 1, Second Floor`, respectfully.
```

     to the :doc:`../../general/users/manage_users` document.

- :guilabel:`Attendance/Point of Sale/Manufacturing`: the employee's :guilabel:`PIN Code` and
  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.

  The :guilabel:`PIN Code` is used to sign in and out of the *Attendance* app kiosk, and a
  :abbr:`POS (Point Of Sale)` system.
- :guilabel:`Payroll`: if applicable, enter the :guilabel:`Registration Number of the Employee` in
  this section.

  Depending on the localization setting, the other items that appear in this field vary based on
  location. In addition, other sections may appear in this tab based on location. It is recommended
  to check with the payroll and/or accounting departments to ensure this section, as well as any
  other sections relating to payroll that may appear, are filled in correctly.
- :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target` for the
  billing rate leader board in the *Timesheets* application. Next, enter the :guilabel:`Hourly Cost`

>   in a XX.XX format. This is factored in when the employee is working at a
>   :doc:`work center <../../inventory_and_mrp/manufacturing/management/using_work_centers>`.

Reviewer 2: Early break
```suggestion
  in a XX.XX format. This is factored in when the employee is working at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`.
```

   :align: center

>    :alt: A skill form with the information filled out.

Author: That is because in that particular Runbot, those are the skill types that were added/created to the skill type list. This will appear different based on how the DB was configured. A blank DB won't have any. This pic is from my DB, so that's why the skills you see are different. So this pic is OK in my opinion since it will always appear different for everyone, based on the skill types (which do not come re-configured)

- :guilabel:`Skill Type`: select a :ref:`skill type <employees/skill-types>` by clicking the radio
  button next to the skill type.
- :guilabel:`Skill`: after selecting a :guilabel:`Skill Type`, the corresponding skills associated
  with that selected :guilabel:`Skill Type` appears in a drop-down menu. For example, selecting
  :guilabel:`Language` as the :guilabel:`Skill Type` presents a variety of languages to select from
  under the :guilabel:`Skills` field. Select the appropriate pre-configured skill, or type in a new

>   skill, then click :guilabel:`Create "(new skill)"`.

Author: Thanks for catching this! Turns out, it's a totally different behavior when there are NO skill types configured, vs when there are.  I redid this section so it is correct and accurate for BOTH possibilities.

delete the entry. Add a new line by clicking the :guilabel:`Add` button next to the corresponding
section.

.. important::
   Only user's with :guilabel:`Officer: Manage all employees` or :guilabel:`Administrator` rights
   for the *Employees* app can add or edit skills.

.. _employees/skill-types:

> Skill Types

Author: Thanks for the suggestion! I think I will leave it here, only because I do like to go in the order of the steps as I would do it/in a logical order on the form. I understand your logic, too, but since I am using this specific 'roadmap' if you will (top info, tab info, each tab explained in detail) I will leave this here. I did add another custom anchor, and now that the info has been updated in the skills section I think it makes more sense. So when someone is doing a new employee form, and they do have the skill types all set, this is a much better flow and makes more sense, as they can skip that part.

types needed.

- :guilabel:`Skill Type`: enter the name of the skill type. This acts as the parent category for
  more specific skills and should be generic.
- :guilabel:`Skills`: click :guilabel:`Add a line` and enter the name for the new skill, then repeat
  for all other needed skills.
- :guilabel:`Levels`: click :guilabel:`Add a line` and enter the name of the level. Next, click into
  the :guilabel:`Progress` field and enter a percentage (0-100) for that level. Repeat for all
  additional levels as needed.
- :guilabel:`Default Level`: click the toggle on the level line to set that level as the default.
  Typically, the lowest level is set as the default, but any level can be chosen. The toggle turns
  green, indicating it is the default level for the skill. Only one level can be set as the default.

Once the form is completely filled out, click the :guilabel:`Save manually` icon at the top of the
screen. Once saved, the :guilabel:`Levels` rearrange in descending order, with the highest level at
the top, and the lowest at the bottom, regardless of the default level and the order they were

> entered.

Author: Correct- there is no need to save manually, but if you do, the levels rearrange, which is why I mentioned it. I updated this to be more accurate.

  .. example::

>      To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the

Author: Thanks for catching that- no save button.no need to save. The save manually is just to rearrange the levels, which is in a tip admonition now. Also, the image is very hard to capture- I tried to make it as large as possible, while still seeing all the information (very hard) so this is as big as I can make it and capture it. I think it's a good example to see how the form looks when filled out, and since I call out all the words/entries I think it's OK. Unless someone has a larger monitor and can capture the image for me, I don't know how I could make it better/sharper. I have this issue with a lot of the larger forms tat take up a big chunk of the screen.

     location, then make any changes on the form. Click :guilabel:`New` to create a new location,
     then enter the following information on the form. All fields are required.

     - :guilabel:`Work Location`: enter the name for the location. This can be as general or
       granular as needed, such as `Home` or `Building 1, Second Floor`, respectfully.
     - :guilabel:`Work Address`: using the drop-down menu, select the address for the location.
     - :guilabel:`Cover Image`: click on the icon to select it for the :guilabel:`Cover Image`.
       Options are a house icon, an office building icon, and a GPS location marker.
     - :guilabel:`Company`: using the drop-down menu, select the company the location applies to.
       The current company populates this field by default.

     .. image:: new_employee/location.png
        :align: center
        :alt: A new work location form with all fields filled out.

- :guilabel:`Schedule`: select the :guilabel:`Working Hours` and :guilabel:`Timezone` for the
  employee. The :guilabel:`Internal Link` arrow opens up a detailed view of the specific daily
  working hours. Working hours can be modified or deleted here.

>   .. note::

Author: Hi @jero-odoo - I can't be 100% sure what the edits requested is here. I might need to talk to you in the AM to get some clarification.

>   .. note::

Author: Closing this since we discussed this edit this morning.

     - In order for the user to appear as an approver for :guilabel:`Time Off`, they must have
       either :guilabel:`Officer` or :guilabel:`Administrator` set for the :guilabel:`Time Off`
       role.
     - In order for the user to appear as an approver for :guilabel:`Timesheets`, they must have
       either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
       :guilabel:`Payroll` role.

- :guilabel:`Remote Work`: using the drop-down menu, select the default location the employee works
  from, for each day of the week. The default options are :guilabel:`Home`, :guilabel:`Office`, or
  :guilabel:`Other`. A new location can be typed into the field, then click either :guilabel:`Create
  (new location)` to add the location, or :guilabel:`Create and edit...` to add the new location and
  edit the form. After edits are done, click :guilabel:`Save & Close`, and the new location is
  added, and populates the field. For days the employee does not work, for example, Saturday and
  Sunday, leave the field blank (:guilabel:`Unspecified`).

  .. note::
     It is also possible to add or modify work locations by navigating to :menuselection:`Employees

>      app --> Configuration --> Employee: Work Locations`. To modify a location, click on an existing

Reviewer 3: I can't comment on this line, but on 230 - 231, in the important admonition block
"To check who has these rights, go to..." There's an extra arrow in the menuselection navigation
![image](https://github.com/odoo/documentation/assets/23611711/8c45d593-5e8f-4fd3-9aad-1b7e389532af)

Author: The extra arrow is because what you click on is actually the arrow with the words Manage Users. It's odd, but if you go into the settings, you'll see it.

  working hours. Working hours can be modified or deleted here.

  .. note::
     :guilabel:`Working Hours` are related to a company's working schedules, and an employee cannot
     have working hours that are outside of a company's working schedule.

     Each individual working schedule is company-specific, so for multi-company databases, each
     company needs to have its own working schedules set.

     If an employee's working hours are not configured as a working schedule for the company, new
     working schedules can be added, or existing working schedules can be modified.

     Working hours can be modified in the *Payroll* application, where they are referred to as
     :guilabel:`Working Schedules`. For more information on how to create or modify
     :guilabel:`Working Schedules` in the *Payroll* application, refer to the
     :doc:`../../hr/payroll` documentation.

- :guilabel:`Planning`: select a role from the drop-down menu for both the :guilabel:`Roles` and
  the :guilabel:`Default Role` fields. If the :guilabel:`Default Role` is selected as a role, it is
  automatically added to the list of :guilabel:`Roles`.

  .. important::

>      The :guilabel:`Planning` section affects the *Planning* application and will only appear if the
>      *Planning* application is installed. If an employee has assigned :guilabel:`Roles`, they will
>      **only** be assigned to shifts for these roles. If this field is blank, they will be assigned
>      shifts regardless of the role. The :guilabel:`Default Role` will have precedence over the other
>      roles when assigning shifts.

Reviewer 3: ```suggestion
     The :guilabel:`Planning` section, relevant to the *Planning* application, is visible only if the
     app is installed. If an employee has assigned :guilabel:`Roles`, they will
     **only** be assigned shifts for these roles. If this field is blank, they will be assigned
     shifts regardless of the role. The :guilabel:`Default Role` takes precedence when assigning shifts.
```

     either :guilabel:`Manager`, :guilabel:`Officer`, or :guilabel:`Administrator` set for the
     :guilabel:`Payroll` role.

>   .. note::

Author: Yes, and I shortened/combined them. Good idea!

order to properly process payslips and ensure all deductions are accounted for, the employee's
personal information should be entered.

Here, the employee's :guilabel:`Private Contact`, :guilabel:`Family Status`, :guilabel:`Emergency`
contact, :guilabel:`Education`, :guilabel:`Work Permit`, and :guilabel:`Citizenship` information is
entered. Fields are entered either using a drop-down menu, activating a check box, or typing in the
information.

> - :guilabel:`Private Contact`: first, enter the :guilabel:`Private Address` for the employee. Next,
>   enter the employee's :guilabel:`Email` address and :guilabel:`Phone` number in the respective
>   fields. Then enter the employee's :guilabel:`Bank Account Number` using the drop-down menu. If the
>   bank is not already configured (the typical situation when creating a new employee) enter the bank
>   account number, and click :guilabel:`Create and edit..`. A :guilabel:`Create Bank Account Number`
>   form loads. Fill in the information, then click :guilabel:`Save & Close`. Next, select the
>   employee's preferred :guilabel:`Language` from the drop-down menu. Then enter the
>   :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is

Reviewer 3: optional rewording suggestions and starting new paragraph when addressing a new subfield. Totally up to your own personal writing style!
```suggestion
- :guilabel:`Private Contact`: Enter the :guilabel:`Private Address`, :guilabel:`Email`, and
  :guilabel:`Phone` for the employee. Then enter the employee's :guilabel:`Bank Account Number`
  using the drop-down menu.

  If the bank is not configured, enter the bank account number, and click :guilabel:`Create and
  edit..`. In the :guilabel:`Create Bank Account Number` form, fill in the information, and click
  :guilabel:`Save & Close`.

  Next, select the employee's preferred :guilabel:`Language` from the drop-down menu. Then enter the
  :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
```

  Number` form loads. Fill in the information, then click :guilabel:`Save & Close`.

  Next, select the employee's preferred :guilabel:`Language` from the drop-down menu. Then enter the
  :guilabel:`Home-Work Distance` in the field. This field is only necessary if the employee is
  receiving any type of commuter benefits. Lastly, enter the employee's license plate information in
  the :guilabel:`Private Car Plate` field.
- :guilabel:`Family Status`: select the current :guilabel:`Marital Status` using the drop-down menu,
  either :guilabel:`Single`, :guilabel:`Married`, :guilabel:`Legal Cohabitant`, :guilabel:`Widower`,
  or :guilabel:`Divorced`. If the employee has any dependent children, enter the :guilabel:`Number
  of Dependent Children` in the field.
- :guilabel:`Emergency`: type in the :guilabel:`Contact Name` and :guilabel:`Contact Phone` number
  of the employee's emergency contact in the respective fields.
- :guilabel:`Education`: select the highest level of education completed by the employee from the
  :guilabel:`Certificate Level` drop-down menu. Default options include :guilabel:`Graduate`,

>   :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
>   :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
> - :guilabel:`Work Permit`: if the employee has a work permit, enter the information in this section.
>   Type in the :guilabel:`Visa No` (visa number) and/or :guilabel:`Work Permit No` (work permit
>   number) in the corresponding fields. Using the calendar selector, select the :guilabel:`Visa
>   Expiration Date` and/or the :guilabel:`Work Permit Expiration Date` to enter the expiration
>   date(s). If available, upload a digital copy of the :guilabel:`Work Permit` document. Click
>   :guilabel:`Upload your file`, navigate to the work permit file in the file explorer, and click
>   :guilabel:`Open`.

Reviewer: ```suggestion
  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, :guilabel:`Other`, or
  :guilabel:`Master:Civil Engineering`.
  
  Type in the :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective
  fields.
- :guilabel:`Work Permit`: if the employee has a work permit, enter the information in this section.
  Type in the :guilabel:`Visa No` (visa number), and/or :guilabel:`Work Permit No` (work permit
  number), in the corresponding fields.
  
  Using the calendar selector, select the :guilabel:`Visa Expiration Date`, and/or the
  :guilabel:`Work Permit Expiration Date`, to enter the expiration date(s).
  
  If available, upload a digital copy of the :guilabel:`Work Permit` document. Click
  :guilabel:`Upload your file`, navigate to the work permit file in the file explorer, and click
  :guilabel:`Open`.
```

Author: The Master:Civil Engineering only appears in Runbots, and it's data they added/imported for whatever testing reasons. If you make a blank database with all the HR apps (and the relevant ones needed, documents, accounting, planning, etc) that does not appear, and what is in the original list are all that show. That's the reason I did not add it.

- :guilabel:`Citizenship`: this section contains all the information relevant to the citizenship of
  the employee. Some fields use a drop-down menu, as the :guilabel:`Nationality (Country)`,

>   :guilabel:`Gender`, and :guilabel:`Country of Birth` fields do. The :guilabel:`Date of Birth` uses
>   a calendar selector to select the date. First, click on the name of the month, then the year, to
>   access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
>   navigate to the correct year range, and click on the year. Next, click on the month. Last, click
>   on the day to select the date. Type in the information for the :guilabel:`Identification No`

Reviewer: ```suggestion
  :guilabel:`Gender`, and :guilabel:`Country of Birth` fields do.
  
  The :guilabel:`Date of Birth` uses a calendar selector to select the date. First, click on the
  name of the month, then the year, to access the year ranges. Use the :guilabel:`< (left)` and
  :guilabel:`> (right)` arrow icons, navigate to the correct year range, and click on the year. Next,
  click on the month. Last, click on the day to select the date.
  
  Type in the information for the :guilabel:`Identification No`
```

  (identification number), :guilabel:`Passport No` (passport number), and :guilabel:`Place of Birth`

>   fields. Last, if the employee is not a resident of the country they are working in, activate the

Reviewer: ```suggestion
  fields.
  
  Lastly, if the employee is **not** a resident of the country they are working in, activate the
```

  :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
  :guilabel:`Freelancer`.

  .. important::

>      Employees do not also need to be users. *Employees* do **not** count towards the Odoo

Reviewer 3: No need to format employees, unless it's a field. In that case, a guilabel is used here
```suggestion
     Employees do not also need to be users. Employees do **not** count towards the Odoo
```

Author: I wanted to emphasize the 'employees' vs 'users' here, so that's why I chose italics.

     Then, enter the :guilabel:`Phone` and :guilabel:`Mobile` numbers in the respective fields. If a
     photo is available, click the :guilabel:`Edit` icon (which appears as a :guilabel:`✏️ (pencil)`
     icon) in the lower left corner of the image box in the top right of the form. A file explorer
     pops up. Navigate to the file, then click :guilabel:`Open` to select it. Finally, click
     :guilabel:`Save` after all the information is entered, and the employee record is automatically
     updated with the newly created user populating the :guilabel:`Related User` field.

     Users can also be created manually. For more information on how to manually add a user, refer
     to the :doc:`../../general/users/manage_users` document.

- :guilabel:`Attendance/Point of Sale/Manufacturing`: the employee's :guilabel:`PIN Code` and
  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID. The :guilabel:`PIN
  Code` is used to sign in and out of the *Attendance* app kiosk, and a :abbr:`POS (Point Of Sale)`
  system.

> - :guilabel:`Payroll`: if applicable, enter the :guilabel:`Registration Number of the Employee` in
>   this section. Depending on the localization setting, the other items that appear in this field
>   vary based on location. In addition, other sections may appear in this tab based on location. It
>   is recommended to check with the payroll and/or accounting departments to ensure this section, as
>   well as any other sections relating to payroll that may appear, are filled in correctly.
> - :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target`. This only
>   affects the billing rate leader board in the *Timesheets* application. Next, enter the
>   :guilabel:`Hourly Cost` in a XX.XX format. This is factored in when the employee is working at a
>   :doc:`work center <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. This
>   value affects the manufacturing costs for a product, if the value of the manufactured product is
>   not a fixed amount. This value does not affect the *Payroll* application. If applicable, enter the
>   :guilabel:`Fleet Mobility Card` number.

Reviewer 3: I'd probably put the other manufacturing cost details in a note block. And I tried to help with brevity in this paragraph. It's all optional, so feel free to change/ignore it (:

```suggestion
- :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target`for the
  billing rate leader board in the *Timesheets* application. Next, enter the :guilabel:`Hourly Cost`
  in a decimal format, which affects manufacturing costs at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. If applicable, enter the
  :guilabel:`Fleet Mobility Card` number.

  .. note::
     Manufacturing costs are added to the costs for producing a product, if the value of the
     manufactured product is not a fixed amount. This cost does not affect the *Payroll*
     application.
```

>   :guilabel:`Fleet Mobility Card` number.

Reviewer 3: Optional suggestion for the last paragraph:
```rst
All employee-related documents are stored in the *Documents* app. The number of associated documents
is displayed in the :guilabel:`Documents` smart button above the employee record. Click on the smart
button to access all documents. Refer to :doc:`documentation <../../finance/documents>` on the
*Documents* app for more information.
```

  :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
  :guilabel:`Freelancer`.

  .. important::
     Employees do not also need to be users. *Employees* do **not** count towards the Odoo
     subscription billing, while *Users* **do** count towards billing. If the new employee should

>      also be a user, the user must be created. After the :guilabel:`User` is created and saved, the
>      new user will appear in the :guilabel:`Related User` field.

Reviewer 4: ```suggestion
     also be a user, the user must be created.
```

>      After the employee is created, create the user. Click the :guilabel:`⚙️ (gear)` icon, then

Reviewer 4: ```suggestion
     After the employee is created, click the :guilabel:`⚙️ (gear)` icon, then
```

  :guilabel:`Employee`, :guilabel:`Student`, :guilabel:`Trainee`, :guilabel:`Contractor`, or
  :guilabel:`Freelancer`.

  .. important::

>      Employees do not also need to be users. *Employees* do **not** count towards the Odoo
>      subscription billing, while *Users* **do** count towards billing. If the new employee should
>      also be a user, the user must be created.
> 
>      After the employee is created, click the :guilabel:`⚙️ (gear)` icon, then click
>      :guilabel:`Create User`. A :guilabel:`Create User` form appears. Type in the :guilabel:`Name`
>      and :guilabel:`Email Address`. Next, select the :guilabel:`Company` from the drop-down menu.
>      Then, enter the :guilabel:`Phone` and :guilabel:`Mobile` numbers in the respective fields. If a
>      photo is available, click the :guilabel:`Edit` icon (which appears as a :guilabel:`✏️ (pencil)`
>      icon) in the lower left corner of the image box in the top right of the form. A file explorer
>      pops up. Navigate to the file, then click :guilabel:`Open` to select it. Finally, click
>      :guilabel:`Save` after all the information is entered, and the employee record is automatically
>      updated with the newly created user populating the :guilabel:`Related User` field.
> 
>      Users can also be created manually. For more information on how to manually add a user, refer
>      to the :doc:`../../general/users/manage_users` document.
> 
> - :guilabel:`Attendance/Point of Sale/Manufacturing`: the employee's :guilabel:`PIN Code` and
>   :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
>   :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID. The :guilabel:`PIN
>   Code` is used to sign in and out of the *Attendance* app kiosk, and a :abbr:`POS (Point Of Sale)`
>   system.

Reviewer: ```suggestion
     Employees do **not** also need to be users. *Employees* do **not** count towards the Odoo
     subscription billing, while *Users* **do** count towards billing. If the new employee should
     also be a user, the user **must** be created.

     After the employee is created, click the :guilabel:`⚙️ (gear)` icon, then click
     :guilabel:`Create User`. A :guilabel:`Create User` form appears.
     
     Type in the :guilabel:`Name` and :guilabel:`Email Address`. Next, select the
     :guilabel:`Company` from the drop-down menu.
     
     Then, enter the :guilabel:`Phone` and :guilabel:`Mobile` numbers in the respective fields.
     
     If a photo is available, click the :guilabel:`Edit` icon (which appears as a :guilabel:`✏️
     (pencil)` icon) in the lower-left corner of the image box, which is located in the top-right of
     the form.
     
     A file explorer pops up. Navigate to the file, then click :guilabel:`Open` to select it.
     Finally, click :guilabel:`Save` after all the information is entered, and the employee record is
     automatically updated with the newly-created user populating the :guilabel:`Related User` field.

     Users can also be created manually. For more information on how to manually add a user, refer
     to the :doc:`../../general/users/manage_users` document.

- :guilabel:`Attendance/Point of Sale/Manufacturing`: the employee's :guilabel:`PIN Code` and
  :guilabel:`Badge ID` can be entered here, if the employee needs/has one. Click
  :guilabel:`Generate` next to the :guilabel:`Badge ID` to create a badge ID.
  
  The :guilabel:`PIN Code` is used to sign in and out of the *Attendance* app kiosk, and a
  :abbr:`POS (Point Of Sale)` system.
```

> - :guilabel:`Payroll`: if applicable, enter the :guilabel:`Registration Number of the Employee` in
>   this section. Depending on the localization setting, the other items that appear in this field
>   vary based on location. In addition, other sections may appear in this tab based on location. It
>   is recommended to check with the payroll and/or accounting departments to ensure this section, as
>   well as any other sections relating to payroll that may appear, are filled in correctly.
> - :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target` for the
>   billing rate leader board in the *Timesheets* application. Next, enter the :guilabel:`Hourly Cost`
>   in a XX.XX format. This is factored in when the employee is working at a
>   :doc:`work center <../../inventory_and_mrp/manufacturing/management/using_work_centers>`. If
>   applicable, enter the :guilabel:`Fleet Mobility Card` number.
> 
> .. note::
>    Manufacturing costs are added to the costs for producing a product, if the value of the
>    manufactured product is not a fixed amount. This cost does not affect the *Payroll* application.

Reviewer: ```suggestion
- :guilabel:`Payroll`: if applicable, enter the :guilabel:`Registration Number of the Employee` in
  this section.
  
  Depending on the localization setting, the other items that appear in this field vary based on
  location. In addition, other sections may appear in this tab based on location. It is recommended
  to check with the payroll and/or accounting departments to ensure this section, as
  well as any other sections relating to payroll that may appear, are filled in correctly.
- :guilabel:`Application Settings`: enter the employee's :guilabel:`Billing Time Target` for the
  billing rate leader board in the *Timesheets* application. Next, enter the :guilabel:`Hourly Cost`
  in a `XX.XX`` format. This is factored in when the employee is working at a :doc:`work center
  <../../inventory_and_mrp/manufacturing/management/using_work_centers>`.
  
  If applicable, enter the :guilabel:`Fleet Mobility Card` number.

.. note::
   Manufacturing costs are added to the costs for producing a product, if the value of the
   manufactured product is **not** a fixed amount. This cost does **not** affect the *Payroll*
   application.
```

.. image:: new_employee/hr-settings.png
   :align: center
   :alt: Enter any information prompted in the HR Settings tab for the employee.

.. _employees/docs:

Documents
=========

All employee-related documents are stored in the *Documents* app. The number of associated documents
is displayed in the :guilabel:`Documents` smart button above the employee record. Click on the smart

> button to access all documents. Refer to :doc:`documentation <../../finance/documents>` on the
> *Documents* app for more information.

Reviewer: ```suggestion
button to access all documents.

Refer to :doc:`documentation <../../finance/documents>` on the *Documents* app for more information.
```
